docker/files/cli/commands/cluster.sh

```diff
@@ -72,7 +72,7 @@ command_remove() {
 commands_add "sync:default-hostgroup" "Synchronize the default hostgroup from backends to the cluster"
 command_sync:default-hostgroup() {
 
-    proxysql_wait_for_admin
+    proxysql_wait_for_admin_job
```

Comment:
Shouldn't we use both? proxysql_wait_for_admin to wait for the settings to be loaded an proxysql_wait_for_admin_job te check if the cluster is up.

Reply:
`sync:default-hostgroup` is running in a cronjob pod where `PROXYSQL_SERVICE` can not be accessed by `127.0.0.1`。

Reply:
You are right, my mistake. Thanks for your contribution!